src/NHibernate/Type/TypeHelper.cs

```diff
@@ -342,4 +342,4 @@ private static bool Dirty(StandardProperty[] properties, object[] currentState,
 			}
 		}
 	}
-}
```

Comment:
Avoid, if possible

Reply:
Is this problem with my IDE and line endings or maybe file encoding.

Reply:
The .editorconfig mandates a blank end line, and enforces this on all changed files. So when reverting a file, do not just manually revert it, but reset it.